src/services/api.service.ts

```diff
@@ -568,6 +580,13 @@ class ApiService {
     return this.$http.post(this.apisURL + api.id + '/reviews?action=REJECT', { message: message }, { headers: { 'If-Match': api.etag } });
   }
 
+  /*
+   * Api Keys
+   */
+  verifyApiKey(apiId: string, apiKey: string): ng.IPromise<any> {
+    return this.$http.get(this.apisURL + apiId + '/keys/_verify?apiKey=' + apiKey);
```

Comment:
post instead of get (see comment in rest-api PR)

Reply:
resolved